app/models/releases/train.rb

```diff
@@ -39,6 +39,7 @@ class Releases::Train < ApplicationRecord
   has_many :deployments, through: :steps
 
   scope :running, -> { includes(:runs).where(runs: {status: Releases::Train::Run.statuses[:on_track]}) }
+  scope :only_with_runs, -> { joins(:runs).where.not(runs: {completed_at: nil}).distinct }
```

Comment:
so, this will not show trains which has a run on track only. is that okay? I think we need to say `.where.not(status: "stopped")`, no?